src/qt/pivx.qrc

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE RCC><RCC version="1.0">
+<RCC>
```

Comment:
was this something your IDE did on it's own? If not why was this changed?

Reply:
hmmm... i think it may have automatically changed it when i added an icon that I have since deleted.